outputs/json/index.ts

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation and others.
+// SPDX-License-Identifier: Apache-2.0
+
+import OutputRenderer from '../base';
+import { LicenseBucket } from '../../structure';
+
+/*
+ * This is used to facilitate chaining across different toolsets
+ */
+export default class JsonRenderer implements OutputRenderer<any> {
```

Comment:
I'm starting to have doubts about this one. If this is going to be a `JsonRenderer` should the result be stringified? 
The caller would then have to `JSON.parse()` to start adding additional packages. 

Thoughts?

Reply:
Eh, I think it's fine as-is -- kinda like how JavaScript toJSON things return a JSON-_capable_ object, instead of a string directly.